plugins/woocommerce/src/Internal/Utilities/DatabaseUtil.php

```diff
@@ -357,7 +357,57 @@ public function create_fts_index_order_address_table(): void {
 		global $wpdb;
 		$address_table = $wpdb->prefix . 'wc_order_addresses';
 		// phpcs:ignore WordPress.DB.PreparedSQL.InterpolatedNotPrepared -- $address_table is hardcoded.
-		$wpdb->query( "CREATE FULLTEXT INDEX order_addresses_fts ON $address_table (first_name, last_name, company, address_1, address_2, city, state, postcode, country, email)" );
+		$wpdb->query( "CREATE FULLTEXT INDEX order_addresses_fts ON $address_table (first_name, last_name, company, address_1, address_2, city, state, postcode, country, email, phone)" );
+	}
+
+	/**
+	 * Helper method to drop the fulltext index on order address table.
+	 *
+	 * @since 9.4.0
+	 *
+	 * @return void
+	 */
+	public function drop_fts_index_order_address_table(): void {
+		global $wpdb;
+		$address_table = $wpdb->prefix . 'wc_order_addresses';
+		// phpcs:ignore WordPress.DB.PreparedSQL.InterpolatedNotPrepared -- $address_table is hardcoded.
+		$wpdb->query( "ALTER TABLE $address_table DROP INDEX order_addresses_fts;" );
+	}
+
+	/**
+	 * Sanitize FTS Search params to remove relevancy operators for performance, and add partial matches. Useful when the sorting is already happening based on some other conditions, so relevancy calculation is not needed.
+	 *
+	 * @since 9.4.0
+	 *
+	 * @param string $param Search term.
+	 *
+	 * @return string Sanitized search term.
+	 */
+	public function sanitise_boolean_fts_search_term( string $param ): string {
```

Comment:
🤔 I don't understand why "boolean" is part of the name of this function...

Reply:
So boolean is the mode of search we are doing. Essentially, we can also search in NATURAL LANGUAGE mode, but that doesn't allow partial searches, so we search in boolean mode instead. See [boolean full text searches](https://dev.mysql.com/doc/refman/8.4/en/fulltext-boolean.html).

This method adds sanitization for boolean search mode, that's why I added the name,